comments/views.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework.generics import ListCreateAPIView, RetrieveUpdateDestroyAPIView
+
+from foods.views import CategoryDetail
+from .models import Comment
+from .serializers import CommentSerializer, CommentDetailSerializer
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+from rest_framework import status
+
+
+class CommentList(ListCreateAPIView):
+    serializer_class = CommentSerializer
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def perform_create(self, serializer):
+        serializer.save(user_id=self.request.user)
+
+    def list(self, request, *args, **kwargs):
+        food_id = self.request.query_params.get("food_id")
+        queryset = Comment.objects.filter(food_id=food_id)
+        serializer = CommentDetailSerializer(queryset, many=True)
```

Comment:
why do you CommentDetailSerializer in list?

Reply:
use to return a detailed list, instead of the list of information in the comments table